keras/src/backend/__init__.py

```diff
@@ -47,3 +48,8 @@
     distribution_lib = None
 else:
     raise ValueError(f"Unable to import backend : {backend()}")
+
+
+@keras_export("keras.Variable")
```

Comment:
Please delete `exports.py` and move all the exports to the end of the file here, that's cleaner. No need to make Variable an exception

Reply:
Done.